backend/src/routes/user/users.router.ts

```diff
@@ -0,0 +1,14 @@
+import * as express from 'express'
+export const userRouter = express.Router()
+
+import {
+  getUser,
+  registerUser,
+  userRegisterPage,
+  userLoginPage,
+  loginUser,
+} from './users.controller'
+
+userRouter.get('/', getUser)
+userRouter.route('/register').get(userRegisterPage).post(registerUser)
+userRouter.route('/login').get(userLoginPage).post(loginUser)
```

Comment:
I think there is no need for get requests on these routes when user logs in or registers we send data back with the response if required, feel free to share another way I may not be aware of :)

Reply:
The Get request will serve the login and register pages

Reply:
I don't think we need to serve these pages because the app will be an SPA. can't hurt to keep them in there for now though

Reply:
The backend will only be concerned with managing data. All entire apps Ul will be served on first request